app/views/posts/all_posts.html.erb

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Your views should have  corresponding view files for each of the controller actions please update your view file names for both post and user views. they should look like this :point_down: 

```ruby
    # app/views/users/index.html.erb
    # app/views/users/show.html.erb

    # app/views/posts/index.html.erb
    # app/views/posts/show.html.erb
```

Reply:
This issue is well resolved! ✔️